helm/prometheus-chart/values.yaml

```diff
@@ -868,7 +868,7 @@ grafana:
   affinity: {}
 
   adminUser: admin
-  # adminPassword: strongpassword
+  adminPassword: Ericom123$
```

Comment:
Errr what?

Reply:
```suggestion
  adminPassword: strongpassword
```
It is a suggestion message, so let's people decide which one use